client/src/Components/RecordingsBrowser/RecordingsBrowser.jsx

```diff
@@ -35,6 +45,7 @@ function GroupByFolder(files, root) {
       if (isAFile) {
         currentFolder.contents.push({
           ...file,
+          relativeKey: relativeKey,
```

Comment:
why not keep the file.relativeKey being instantiated as before?

Reply:
Dont know will check :) dont tremember why I did this change anymore 😆

Reply:
Ok remember now it was not working for local folders becaue you needed to have the relative key as part of the current folder information.